src/utilities/scroll-lock-manager/scroll-lock-manager.ts

```diff
@@ -1,8 +1,6 @@
 import {isServer} from '../target';
 
-export const SCROLL_LOCKING_ATTRIBUTE = 'data-lock-scrolling';
-export const SCROLL_LOCKING_WRAPPER_ATTRIBUTE = 'data-lock-scrolling-wrapper';
-export const SCROLL_LOCKING_CUSTOM_PROPERTY = '--scroll-lock-body-padding';
```

Comment:
I couldn't find any usage of this custom property, so I deleted it. Please let me know if I missed something.